bookList.js

```diff
@@ -4,12 +4,12 @@ class Booklist {// eslint-disable-line no-unused-vars
   }
 
   addBook(title, author, id) {
-    this.books.concat({ title, author, id });
+    this.books.push({ title, author, id });
     localStorage.setItem('books', JSON.stringify(this.books));
     return { title, author, id };
   }
 
-  removeBook(element, id) {
+  removeBook(element, id, parent) {
     if (element.classList.contains('remove')) {
       const removeItem = element.parentElement;
       this.books = this.books.filter((book) => {
```

Comment:
- [ OPTIONAL] There is a cleaner way to write this filter function by returning the explicit comparison result. Feel free to refactor it 🌝.
` this.books = this.books.filter((book) => book.id !== id)`

Reply:
Thanks for mentioning it. :+1: